service/src/test/resources/ehr/mapper/encountercomponents/expected-components-1.xml

```diff
@@ -50,38 +51,46 @@
             <name code="394847000" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Unspecified significance"/>
         </qualifier>
     </code>
+    <code nullFlavor="UNK">
+        <originalText>4</originalText>
+    </code>
 </LinkSet>
 </component>
-<component typeCode="COMP">
+<component typeCode="COMP" >
 <ObservationStatement classCode="OBS" moodCode="EVN">
-    <id root="394559384658936"/>
-    <statusCode code="COMPLETE"/>
+    <id root="394559384658936" />
+    <statusCode code="COMPLETE" />
     <effectiveTime>
         <center value="20100630055900"/>
     </effectiveTime>
-    <availabilityTime value="20100630054806"/>
+    <availabilityTime value="20100630054806" />
     <pertinentInformation typeCode="PERT">
         <sequenceNumber value="+1"/>
         <pertinentAnnotation classCode="OBS" moodCode="EVN">
-            <text>immunization note</text>
+            <text>immunization note Substance: unknown</text>
```

Comment:
Should this text actually be prepended to the note? Verify this one.

Reply:
Would be worth mentioning that this is also gone now as UNK is not mapped.

Reply:
Verified by David, render as text element unless unknown.